index.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+class Application
+{
+
+    function run()
+    {
+        $times = 0;
+        $historyRepository = new HistoryRepository();
+        $date = date('d-m-Y');
```

Comment:
not found where date variable is using

Reply:
historyRepository -> create ($date ... )